src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/MethodLengthCheck.java

```diff
@@ -73,26 +73,127 @@
+ *       // line 2
+ *       System.out.println("line 3");
+ *
+ *       /&#42; line 5
+ *           ...
+ *          line 150 &#42;/
+ *   } // line 151, violation, as it is over 150
+ * }
+ * </pre>
+ * <p>
+ * To configure the check so that it accepts methods with at most 4 lines:
  * </p>
  * <pre>
  * &lt;module name="MethodLength"&gt;
  *   &lt;property name="tokens" value="METHOD_DEF"/&gt;
- *   &lt;property name="max" value="60"/&gt;
+ *   &lt;property name="max" value="4"/&gt;
  * &lt;/module&gt;
  * </pre>
  * <p>
- * To configure the check so that it accepts methods with at most 60 lines,
+ * Example:
+ * </p>
+ * <pre>
+ * public class MyTest {
+ *   public MyTest()  {  // constructor (line 1)
+ *       int a = 2; // line 2
+ *       int b = 4; // line 3
+ *       int c = a + b; // line 4
+ *   } // line 5, OK, constructor is not mentioned in the tokens
+ *
+ *   public void firstMethod() { // line 1
+ *       // comment (line 2)
+ *       System.out.println("line 3");
+ *   } // line 4, OK, as it allows at most 4 lines
+ *
+ *   public void secondMethod() { // line 1
+ *       int i = 0;    // line 2
+ *       if (i &#60; 5) { // line 3
+ *           i++;      // line 4
+ *       }             // line 5
+ *   } // line 6, violation, as it is over 4 lines
+ *
+ *   public void thirdMethod() { // line 1
+ *
+ *       // comment (line 3)
+ *       System.out.println("line 4");
+ *   } // line 5, violation, as it is over 4 lines
+ *
+ *   public void fourthMethod() { // line 1
+ *       /&#42; line 2
+ *          line 3
+ *          line 4 &#42;/
+ *   } // line 5, violation, as it is over 4 lines
+ * }
+ * </pre>
+ * <p>
+ * To configure the check so that it accepts methods with at most 4 lines,
  * not counting empty lines and comments:
  * </p>
  * <pre>
  * &lt;module name="MethodLength"&gt;
  *   &lt;property name="tokens" value="METHOD_DEF"/&gt;
- *   &lt;property name="max" value="60"/&gt;
+ *   &lt;property name="max" value="4"/&gt;
  *   &lt;property name="countEmpty" value="false"/&gt;
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public class MyTest {
+ *   public MyTest()  {  // constructor (line 1)
+ *       int a = 2; // line 2
+ *       int b = 4; // line 3
+ *       int c = a + b; // line 4
+ *   } // line 5, OK, constructor is not mentioned in the tokens
+ *
+ *   public void firstMethod() { // line 1
+ *       // comment
+ *       System.out.println("line 2");
+ *   } // line 3, OK, as it allows at most 4 lines
+ *
+ *   public void secondMethod() { // line 1
+ *       int i = 0;    // line 2
+ *       if (i &#60; 5) { // line 3
+ *           i++;      // line 4
+ *       }             // line 5
+ *   } // line 6, violation, as it is over 4 lines
+ *
+ *   public void thirdMethod() { // line 1
+ *
+ *       // comment
+ *       System.out.println("line 2");
+ *   } // line 3, OK, as it allows at most 4 lines
+ *
+ *   public void fourthMethod() { // line 1
```

Comment:
behavior on forth is same on countEmpty=false and countEmpty=true

lets remove it, we already showed user behavior on comment in default config.

Reply:
Thank you for the feedback.
I removed the fourth example.